Kitodo/src/main/java/org/kitodo/production/helper/ProcessHelper.java

```diff
@@ -99,7 +100,11 @@ public static ProcessFieldedMetadata initializeProcessDetails(LogicalDivision st
             RulesetManagementInterface managementInterface, String stage, List<Locale.LanguageRange> priorityList) {
         StructuralElementViewInterface divisionView = managementInterface.getStructuralElementView(structure.getType(),
                 stage, priorityList);
-        return new ProcessFieldedMetadata(structure, divisionView);
+        try {
+            return new ProcessFieldedMetadata(structure, divisionView);
+        } catch (NullPointerException e) {
```

Comment:
I think when we catch NPE than we should be more direct on the problem e.g. in function `ProcessFieldedMetadata.createMetadataEntryEdit`. This function should throw `ProcessException` or a new  `ProcessFieldException`. Otherwise it leads to confusion, misinterpretation or hides other problems